scripts/scenarios/cloud6/qa-scenario-2b-sbd-kvm-vmware.yaml

```diff
@@ -131,6 +131,8 @@ proposals:
         host: vcs.qa.suse.de
         user: "@@vcenter_user@@"
         password: "@@vcenter_password@@"
+        cluster_name: []
```

Comment:
> so we have to leave this as empty or allow to pass it as parameter.

@vmoravec any opinion on this? I can add a placeholder -- we can then just change the 2b job in jenkins.